upstream bug with DateTime

```
I'm getting the same exact error with a fresh install ubuntu:

PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-basic.t ........... ok    
t/00-crypto.t .......... ok   
t/00-dates.t ........... 1/? Validation failed for type named DayOfMonth declared in package DateTime::Types (/usr/local/lib/perl/5.18.2/DateTime/Types.pm) at line 29 in sub named (eval) with value 2015

Trace begun at /usr/local/share/perl/5.18.2/Specio/Exception.pm line 43
Specio::Exception::BUILD('Specio::Exception=HASH(0x1492620)', 'HASH(0x222fd40)') called at Specio::Exception->new line 57
Eval::Closure::Sandbox_17::__ANON__('Specio::Exception', 'message', 'Validation failed for type named DayOfMonth declared in package DateTime::Types (/usr/local/lib/perl/5.18.2/DateTime/Types.pm) at line 29 in sub named (eval) with value 2015', 'type', 'Specio::Constraint::Simple=HASH(0x27a01f0)', 'value', 2015) called at /usr/local/share/perl/5.18.2/Specio/Exception.pm line 62
Specio::Exception::throw('Specio::Exception', 'message', 'Validation failed for type named DayOfMonth declared in package DateTime::Types (/usr/local/lib/perl/5.18.2/DateTime/Types.pm) at line 29 in sub named (eval) with value 2015', 'type', 'Specio::Constraint::Simple=HASH(0x27a01f0)', 'value', 2015) called at (eval 303) line 85
DateTime::_check_new_params('year', 1929, 'month', 7, 'day', 2015, 'time_zone', -0700) called at /usr/local/lib/perl/5.18.2/DateTime.pm line 160
DateTime::new('DateTime', 'year', 1929, 'month', 7, 'day', 2015, 'time_zone', -0700) called at /usr/local/share/perl/5.18.2/DateTime/Format/DateParse.pm line 75
DateTime::Format::DateParse::parse_datetime('DateTime::Format::DateParse', '29/7/2015') called at /home/joao/massive-octo-spice/src/blib/lib/CIF/Plugin/DateTime.pm line 98
CIF::Plugin::DateTime::try {...}  at /usr/share/perl5/Try/Tiny.pm line 81
eval {...} at /usr/share/perl5/Try/Tiny.pm line 72
Try::Tiny::try('CODE(0x1492338)', 'Try::Tiny::Catch=REF(0x1491fc0)') called at /home/joao/massive-octo-spice/src/blib/lib/CIF/Plugin/DateTime.pm line 111
CIF::Plugin::DateTime::normalize_timestamp('29/7/2015') called at t/00-dates.t line 28
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 6.
t/00-dates.t ........... Dubious, test returned 255 (wstat 65280, 0xff00)
All 6 subtests passed 
t/00-tokens.t .......... ok   
t/00-urls.t ............ ok    
t/10-client.t .......... ok   
t/10-logger.t .......... 1/? [2016-10-01T15:18:58,326Z][57300][ERROR]: IGNORE: this is another test
t/10-logger.t .......... ok   
t/10-messages.t ........ ok   
t/10-simple_feed.t ..... ok   
t/10-worker.t .......... ok   
t/20-alexa.t ........... ok   
t/20-arborflux-xml.t ... ok   
t/20-bambenek.t ........ ok   
t/20-blocklist_dt.t .... ok   
t/20-isc_sans_edu.t .... ok    
t/20-malc0de.t ......... ok    
t/20-malwaredomains.t .. ok    
t/20-mirc.t ............ ok   
t/20-observable.t ...... ok   
t/20-openbl.t .......... ok   
t/20-openphish.t ....... ok   
t/20-phishtank.t ....... ok   
t/20-ransomware.t ...... ok    
t/20-sslbl_abuse.t ..... ok   
t/20-zeustracker.t ..... ok   
t/50-router.t .......... 2/? # starting client
# starting router...
# waiting...
# running ping...
# killing router...
t/50-router.t .......... ok   
t/60-freeform.t ........ ok   
t/60-passivedns.t ...... ok   

Test Summary Report
-------------------
t/00-dates.t         (Wstat: 65280 Tests: 6 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
Files=28, Tests=235, 18 wallclock secs ( 0.08 usr  0.02 sys + 14.72 cusr  1.36 csys = 16.18 CPU)
Result: FAIL
Failed 1/28 test programs. 0/235 subtests failed.
make[1]: *** [test_dynamic] Error 255
make[1]: Leaving directory `/home/joao/massive-octo-spice/src'
make: *** [test] Error 2

Can someone help ? :)

Thanks in advance
Best Regards!

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,7 +17,7 @@ install:
     - sudo apt-get install -y curl mailutils build-essential git-core automake cpanminus rng-tools openjdk-7-jre-headless libtool pkg-config vim htop bind9 libzmq3-dev libffi6 libmoose-perl libmouse-perl libanyevent-perl liblwp-protocol-https-perl libxml2-dev libexpat-dev python-dev
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
-    - cpanm --wget --mirror https://cpan.metacpan.org DateTIme@1.36
+    - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,8 +17,8 @@ install:
     - sudo apt-get install -y curl mailutils build-essential git-core automake cpanminus rng-tools openjdk-7-jre-headless libtool pkg-config vim htop bind9 libzmq3-dev libffi6 libmoose-perl libmouse-perl libanyevent-perl liblwp-protocol-https-perl libxml2-dev libexpat-dev python-dev
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
-    - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Test2::Suite@0.000058 
+    - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,5 @@
+sudo: required
+dist: trusty
 language: perl
 perl:
     - 5.18
@@ -15,6 +17,7 @@ install:
     - sudo apt-get install -y curl mailutils build-essential git-core automake cpanminus rng-tools openjdk-7-jre-headless libtool pkg-config vim htop bind9 libzmq3-dev libffi6 libmoose-perl libmouse-perl libanyevent-perl liblwp-protocol-https-perl libxml2-dev libexpat-dev python-dev
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
+    - cpanm --wget --mirror https://cpan.metacpan.org DateTIme::@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,6 +17,7 @@ install:
     - sudo apt-get install -y curl mailutils build-essential git-core automake cpanminus rng-tools openjdk-7-jre-headless libtool pkg-config vim htop bind9 libzmq3-dev libffi6 libmoose-perl libmouse-perl libanyevent-perl liblwp-protocol-https-perl libxml2-dev libexpat-dev python-dev
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
+    - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014
     - cpanm --wget --mirror https://cpan.metacpan.org Test2::Suite@0.000058 
     - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,7 +17,7 @@ install:
     - sudo apt-get install -y curl mailutils build-essential git-core automake cpanminus rng-tools openjdk-7-jre-headless libtool pkg-config vim htop bind9 libzmq3-dev libffi6 libmoose-perl libmouse-perl libanyevent-perl liblwp-protocol-https-perl libxml2-dev libexpat-dev python-dev
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
-    - cpanm --wget --mirror https://cpan.metacpan.org DateTIme::@1.36
+    - cpanm --wget --mirror https://cpan.metacpan.org DateTIme@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,6 +18,7 @@ install:
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
     - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
+    - cpanm --wget --mirror https://cpan.metacpan.org Test2::Suite@0.000058 
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,7 +18,7 @@ install:
     - cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
     - cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
     - cpanm --wget --mirror https://cpan.metacpan.org Test::Simple@1.001014
-    - cpanm --wget --mirror https://cpan.metacpan.org Test2::Suite@0.000058 
+    - cpanm --wget --mirror https://cpan.metacpan.org Test2::Suite@0.000055 
     - cpanm --wget --mirror https://cpan.metacpan.org DateTime@1.36
     - cpanm --wget --mirror https://cpan.metacpan.org Regexp::Common
     - cpanm --wget --mirror https://cpan.metacpan.org Moo@1.007000

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,7 +1,8 @@
 Revision history for massive-octo-spice
 
-2.00.01 2016-09-15
+2.00.01 2016-10-3
     - remote fix for malwaredomains url shortners and bulk registrars
+    - upstream bugfix: DateTime - https://github.com/csirtgadgets/massive-octo-spice/issues/451
 
 2.00.00 2016-09-07
     - first stable release

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,6 @@
 Revision history for massive-octo-spice
 
-2.00.01 2016-10-3
+2.00.01 2016-10-04
     - remote fix for malwaredomains url shortners and bulk registrars
     - upstream bugfix: DateTime - https://github.com/csirtgadgets/massive-octo-spice/issues/451
 

--- a/hacking/platforms/ubuntu/ubuntu.sh
+++ b/hacking/platforms/ubuntu/ubuntu.sh
@@ -56,6 +56,7 @@ alias cpanm='cpanm --wget --mirror https://cpan.metacpan.org --skip-installed'
 
 cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
 cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
+cpanm DateTime@1.36
 cpanm Regexp::Common
 cpanm Moo@1.007000
 cpanm Test::Simple@1.001014

--- a/hacking/platforms/ubuntu/ubuntu.sh
+++ b/hacking/platforms/ubuntu/ubuntu.sh
@@ -56,11 +56,11 @@ alias cpanm='cpanm --wget --mirror https://cpan.metacpan.org --skip-installed'
 
 cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
 cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
+cpanm Test::Simple@1.001014
 cpanm Test2::Suite@0.000058
 cpanm DateTime@1.36
 cpanm Regexp::Common
 cpanm Moo@1.007000
-cpanm Test::Simple@1.001014
 cpanm Mouse@2.4.1
 cpanm ZMQ::FFI@0.17
 cpanm --force --notest https://github.com/csirtgadgets/ZMQx-Class/archive/master.tar.gz

--- a/hacking/platforms/ubuntu/ubuntu.sh
+++ b/hacking/platforms/ubuntu/ubuntu.sh
@@ -56,6 +56,7 @@ alias cpanm='cpanm --wget --mirror https://cpan.metacpan.org --skip-installed'
 
 cpanm https://github.com/csirtgadgets/Net-Abuse-Utils/archive/master.tar.gz
 cpanm https://github.com/csirtgadgets/net-abuse-utils-spamhaus/archive/v0.08.tar.gz
+cpanm Test2::Suite@0.000058
 cpanm DateTime@1.36
 cpanm Regexp::Common
 cpanm Moo@1.007000

--- a/src/cpanfile
+++ b/src/cpanfile
@@ -4,6 +4,7 @@ requires 'Hijk','0.19';
 requires 'Encoding::FixLatin';
 requires 'HTML::TableExtract';
 requires 'Test::Exception', '<= 0.43';
+requires 'Test2::Suite', '0.000058';
 requires 'Net::Server::SS::PreFork';
 requires 'CIF::SDK', '>= 2.00_30';
 requires 'File::Slurp','9999.19';

--- a/src/cpanfile
+++ b/src/cpanfile
@@ -16,6 +16,7 @@ requires 'Compress::Snappy';
 requires 'Crypt::Random::Source';
 requires 'Daemon::Control';
 requires 'Data::Visitor::Callback';
+requires 'DateTime', '1.36';
 requires 'DateTime::Format::DateParse';
 requires 'Digest::SHA';
 requires 'File::Type';

--- a/src/cpanfile
+++ b/src/cpanfile
@@ -1,3 +1,4 @@
+requires 'Test::Simple', '1.001014';
 requires 'Gzip::Faster', '0.14';
 requires 'Mozilla::CA';
 requires 'Hijk','0.19';

